src/main/java/uk/gov/cshr/report/repository/databaseManager/DatabaseSqlHelper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.cshr.report.repository.databaseManager;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class DatabaseSqlHelper implements IDatabaseSqlHelper {
+
+
+    @Override
+    public String getCreateTimeBoundPartitionStatement(String partitionName, String parentTableName, String fromTimestamp, String toTimestamp) {
+        return String.format("CREATE TABLE IF NOT EXISTS %s PARTITION OF %s FOR VALUES FROM ('%s') TO ('%s')",
+                partitionName, parentTableName, fromTimestamp, toTimestamp);
```

Comment:
Can we use `PreparedStatement`s instead of `String.format()` for SQL statements, just to be on the safe side?